sys/include/net/netstats.h

```diff
@@ -27,6 +28,27 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Max length of a L2 address
+ */
+#ifndef CONFIG_L2ADDR_MAX_LEN
+#define CONFIG_L2ADDR_MAX_LEN   (8)
+#endif
```

Comment:
As you use `l2util` now, you could re-use `L2UTIL_ADDR_MAX_LEN` instead of redefining yet another L2 address max length ;-).